controller/src/main/java/io/pravega/controller/server/rest/generated/api/ApiException.java

```diff
@@ -1,6 +1,6 @@
 package io.pravega.controller.server.rest.generated.api;
 
-
+@SuppressWarnings("all")
```

Comment:
Since this is generated, does it make sense to add this annotation here?

Reply:
I annotated the generated code so that it will not generate any warnings. We don't want to be tempted to edit it.